torch/csrc/jit/interned_strings.h

```diff
@@ -133,4 +133,8 @@ enum BuiltinSymbol {
 const char * symbolToString(Symbol s);
 Symbol stringToSymbol(const std::string & s);
 
+inline Symbol operator "" _sym(const char * s, unsigned long) {
```

Comment:
What's the protocol for using ""_sym as opposed to adding an interned string?

Reply:
The downside of this custom literal is that it incurs additional run-time cost, but this is 100% fine if we're just building up static containers of symbols (you pay it once anyway). If a symbol only appears in such cases then I think there's no point in polluting the default list, and this is what I did here with e.g. `_sigmoid_backward`.

Reply:
Anything not in the main list also requires a lock acquire and release for stringToSymbol/symbolToString. I don't really care for here because perf doesn't matter, but it is something to keep in mind.